mocurly/endpoints.py

```diff
@@ -963,6 +968,15 @@ def cancel_subscription(self, pk, cancel_info, format=format):
             'canceled_at': current_time().isoformat()
         }), format=format)
 
+    @details_route('PUT', 'reactivate')
+    def reactivate_subscription(self, pk, reactivate_info, format=format):
+        subscription = SubscriptionsEndpoint.backend.get_object(pk)
```

Comment:
To match [the recurly docs](https://docs.recurly.com/subscriptions), this should double check to make sure the subscription is in the `canceled` state.